src/styles/vars.scss

```diff
@@ -0,0 +1,7 @@
+$grey: #616070;
+$blue: #00acdc;
+$black: #060b35;
+$white: #fff;
+$light-grey: #f3f3f3;
+$dark-blue: #34568b;
```

Comment:
Variable names should show their purpose, for example, `$main-color`, as if the palette changes and all things that were dark blue become red you have to rename all of occurrences